arm-customer-insights/2017-04-26/swagger/customer-insights.json

```diff
@@ -0,0 +1,5664 @@
+          }
+        },
+        "description": "Gets an authorization policy in the hub.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the hub."
+          },
+          {
+            "name": "authorizationPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the policy."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Successfully get the authorization policy.",
+            "schema": {
+              "$ref": "#/definitions/AuthorizationPolicyResourceFormat"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CustomerInsights/hubs/{hubName}/authorizationPolicies": {
+      "get": {
+        "tags": [
+          "AuthorizationPolicies"
+        ],
+        "operationId": "AuthorizationPolicies_ListByHub",
+        "x-ms-examples": {
+          "AuthorizationPolicies_ListByHub": {
+            "$ref": "../examples/AuthorizationPoliciesListByHub.json"
+          }
+        },
+        "description": "Gets all the authorization policies in a specified hub.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the hub."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Successfully get all the authorization policies in the hub.",
+            "schema": {
+              "$ref": "#/definitions/AuthorizationPolicyListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CustomerInsights/hubs/{hubName}/authorizationPolicies/{authorizationPolicyName}/regeneratePrimaryKey": {
```

Comment:
Ok. consider changing this in the next API-version revision to make the API cleaner.